.gitignore

```
@@ -3,3 +3,5 @@ build/
 source/_build/
 /.idea/
 /.tx/
+/.idea
+/.tx
```

are these all needed?
I think 
```suggestion
.idea
.tx
```
should do